src/tests/ftest/util/dmg_utils.py

```diff
@@ -215,10 +213,12 @@ def storage_scan(self, verbose=False):
 
         return data
 
-    def storage_format(self, reformat=False, timeout=30):
+    def storage_format(self, reformat=False, timeout=30, verbose=False):
         """Get the result of the dmg storage format command.
 
         Args:
+            verbose (bool): show results of each SCM & NVMe device format
+                operation.
```

Comment:
It's better this description comes at the end of the Args because `verbose` is the last argument.

Reply:
Will fix.